package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@govtechsg/sgds",
+  "name": "sgds-govtech",
   "version": "1.3.23",
   "description": "Design System unites Singapore Government digital services around a common visual language and user experience",
```

Comment:
verison 1.3.23 already exists on npm sgds-govtech. Should bumped up to 1.3.24 to publish

Reply:
bumped to 1.3.24